src/Files.App/Filesystem/RecentItems.cs

```diff
@@ -105,6 +106,7 @@ public async Task UpdateRecentFoldersAsync()
 		public async Task<List<RecentItem>> ListRecentFilesAsync()
 		{
 			return (await Win32Shell.GetShellFolderAsync(QuickAccessGuid, "Enumerate", 0, int.MaxValue)).Enumerate
+				.Where(link => !link.IsFolder)
```

Comment:
Fix: do not show pinned folders (e.g desktop, documents) in recent list, only files